src/app/products/loan-products/loan-product-stepper/loan-product-accounting-step/advanced-accounting-mapping-rule/advanced-accounting-mapping-rule.component.ts

```
@@ -290,8 +317,15 @@ export class AdvancedAccountingMappingRuleComponent implements OnInit {
   getGlAccountData(valueId: number): any {
-    return this.incomeAccountData.find((i: any) => {
+    const glAccountData = [
+      this.incomeAccountData,
+      this.expenseAccountData,
+      this.assetAccountData,
+      this.liabilityAccountData
+    ];
+    return glAccountData.find((i: any) => {
       return i.id === valueId;
     });
+    return null;
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)